app/Resources/SonataAdminBundle/views/CRUD/base_array_macro.html.twig

```diff
@@ -0,0 +1,36 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+{% macro render_array(value, inline) %}
```

Comment:
1) where is used this macro?
2) all twig templates are in `/templates/` not in `app/Resources`

Reply:
This macro is used by sonata to render array in admin list and view, but it's pretty ugly for human then nobody use it because of that. That's why I was thinking to do it. Now if you want render an array in admin list , it will be a nice list or separate with "," without creating a custom template to do the same thing ^^,
Why in app/Resources, because it's an override template and I it's the only way I found to override this template.